src/silly_sprites/opengl/shader_program.hpp

```diff
@@ -26,5 +26,7 @@ namespace sly::gl {
 
         void bind();
         void unbind();
+
+        friend class Renderer;
```

Comment:
where du we put a `friend class`?
i put it at the beginning (beniging) of the class.
we should agree on one way.

Reply:
I always put it at the bottom. But maybe the beginning would be better? I will move it up.

Reply:
I will even put it before the first `private:` label. Maybe it's more obvious there.